src/main/java/net/atomique/ksar/parser/Linux.java

```diff
@@ -156,8 +159,7 @@ public int parse(String line, String[] columns) {
         GraphConfig mygraphinfo = myosconfig.getGraphConfig(checkStat);
         if (mygraphinfo != null) {
           if ("unique".equals(mygraphinfo.getType())) {
-            obj = new Graph(mysar, mygraphinfo, mygraphinfo.getTitle(), line, firstdatacolumn,
-                mysar.graphtree);
+            obj = new Graph(mysar, mygraphinfo, mygraphinfo.getTitle(), line, firstdatacolumn, mysar.graphtree);
```

Comment:
Please refrain from interleaving refactorings and feature commits in the same PR.
Refactoring is good, however it would be nice to have those separated.